API/TreeData/TreeDataMock.cs

```diff
@@ -0,0 +1,149 @@
+using System;
+using System.Collections;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+using MySql.Data;
+using MySql.Data.MySqlClient;
+
+namespace TreeData_CLI

```

Comment:
This file should be in the directory `WebApi/Triggers`.  Rename this file to `TreeTriggers`. Then namespace should be `ForestGEO.WebApi.Triggers.Tree` Be careful when renaming to use `git mv` to preserve git history.

Reply:
Fixed in https://github.com/ForestGeoHack/ForestGEO/pull/12/commits/a936400665e8815bd67d474b006b44b0b214fd84